### PR TITLE
Updated email examples in rule to make it clearer

### PR DESCRIPTION
**Pull request by matt-goldman**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

Email subject: Rules - Do you know how to hand over a project? from @wicksipedia 

> 2. What was changed?

The email examples were improved to make it clear who is handing over to whom, and also to include actions/tasks.

> 3. Did you do pair or mob programming (list names)?

N/A

### CODE REVIEW

File: rules/hand-over/rule.md [hunk 1]
```diff
@@ -117,9 +121,9 @@ Done - As per our meeting the handover has been completed to my satisfaction
 :::
 :::
 ::: good
-Figure: Good example - This handover has lots of URLs and is complete
+Figure: Good example - This handover has a clear designation of responsibilities, lots of URLs and is complete
 :::
 
-If you need to handover only a single task there are more details on [Do you know how to hand over email tasks to others?](/how-to-hand-over-tasks-to-others)
+If you need to hand over only a single task there are more details on [Do you know how to hand over email tasks to others?](/how-to-hand-over-tasks-to-others)
```
Comment (JackDevAU) [inline rules/hand-over/rule.md:127]: Should this be a single word? from `hand over` to `handover`

**Review by JackDevAU — COMMENTED** [review]
Looks good!

**Review by JackDevAU — APPROVED** [review]
lgtm!

File: rules/hand-over/rule.md (hunk 1 above)
Comment (matt-goldman) [reply rules/hand-over/rule.md:127]: Hand over = verb
Handover = noun

**Pull request merged**